lang/ilias_hu.lang

```diff
@@ -152,7 +152,7 @@ administration#:#restoring_missing_objs#:#Elveszett objektumok visszaállítása
 administration#:#restoring_trash#:#Törölt objektumok visszaállítása...
 administration#:#restoring_unbound_objs#:#Kapcsolódás nélküli objektumok és alobjektumok visszaállítása...
 administration#:#restoring#:#Helyreállítás...
-administration#:#reuse_of_loginnames_contained_in_history#:#A felhasználónevek nem használhatók fel újra.
+administration#:#reuse_of_loginnames_contained_in_history#:#A felhasználónevek újra felhasználhatóak.

```

Comment:
I don't speak Hungarian but Google Translate does :-) If Google does right, the old version said "Usernames can't be reused" and your correction says "Usernames can be reused"... Does that mean that the old version was completely wrong? thanks for your help!